### PR TITLE
Submission of Plinko example

### PR DESCRIPTION
**Pull request by HelperWesley**
The Plinko example showcases the physics behavior and how it along with other tools, like the object shake extension, can create a fun and juicy game.

The example features:
-Physics engine 2.0
-Object shake
-Particles
-Simple scoring system
-Leaderboard
-Text input object(For leaderboard name)

The Game zip file:
[PlinkoExampleGame.zip](https://github.com/GDevelopApp/GDevelop-examples/files/9479852/PlinkoExampleGame.zip)

### CODE REVIEW

**Comment by Bouh** [comment]
- `examples/PlinkoExampleGame/assets/LiluoLogoPlinko.png` is the thumbnail, this file have to be named `thumbnail.png`. Otherwise the IDE won't use the image to display it on the homepage or the examples list.
- The file have to be next to the project file (not in an asset folder).
- When you want, you can update this list by adding this new game (use the folder name you made), to add the game on the homepage.

File: examples/PlinkoExampleGame/PlinkoExampleGame.json [hunk 1]
```diff
@@ -0,0 +1,11787 @@
...
+    "build": 99,
+    "major": 4,
+    "minor": 0,
+    "revision": 0
+  },
+  "properties": {
+    "adaptGameResolutionAtRuntime": true,
+    "folderProject": false,
+    "orientation": "landscape",
+    "packageName": "com.example.gamename",
```
Comment (Bouh) [inline examples/PlinkoExampleGame/PlinkoExampleGame.json:13]:
```suggestion
    "packageName": "com.example.pachinko",
```

**Review by Bouh — APPROVED** [review]
- I also saw unused resources by cleaning them.

**Comment by HelperWesley** [comment]
@Bouh I fixed up those things, and I think it's fine to release now.

Do you have any more concerns about it? Can I press "Squash and merge"?

**Comment by Bouh** [comment]
> @Bouh I fixed up those things, and I think it's fine to release now.
> 
> Do you have any more concerns about it? Can I press "Squash and merge"?

No, and yes :)

**Pull request merged**